standard-library.agda-lib

```diff
@@ -1,4 +1,5 @@
-name: standard-library-2.1
+name: standard-library-tbt
 include: src
 flags:
   --warning=noUnsupportedIndexedMatch
+--   --type-based-termination
```

Comment:
Was this meant to be part of this PR? 
AFAICT, this is part of a separate experiment on the main `Agda` line of development, and not relevant (yet!) for `stdlib-v2.1`?
Should the commit which introduced this be reverted?

Reply:
Aarrgh how did this happen?  Sincere apologies, will fix.

Reply:
https://github.com/agda/agda-stdlib/pull/2326